### PR TITLE
Add Philippines telcos

### PR DESCRIPTION
**Pull request by TagaSanPedroAko**

### CODE REVIEW

File: data/brands/shop/telecommunication.json [hunk 1]
```diff
@@ -184,6 +184,18 @@
         "shop": "telecommunication"
       }
     },
+    {
+      "displayName": "Dito",
+      "locationSet": {"include": ["ph"]},
+      "matchNames": ["dito telecommunity", "mislatel"],
+      "tags": {
+        "alt_name": "Dito Telecommunity"
```
Comment (tommylung) [inline data/brands/shop/telecommunication.json]: Missing comma

File: data/brands/shop/telecommunication.json [hunk 2]
```diff
@@ -195,6 +207,18 @@
         "shop": "telecommunication"
       }
     },
+    {
+      "displayName": "Globe",
+      "locationSet": {"include": ["ph"]},
+      "matchNames": ["globe telecom"],
+      "tags": {
+        "alt_name": "Globe Telecom"
```
Comment (tommylung) [inline data/brands/shop/telecommunication.json]: Missing comma

File: data/brands/shop/telecommunication.json [hunk 3]
```diff
@@ -275,6 +299,18 @@
         "shop": "telecommunication"
       }
     },
+    {
+      "displayName": "Smart",
+      "locationSet": {"include": ["ph"]},
+      "matchNames": ["smart communications"],
+      "tags": {
+        "alt_name": "Smart Communications"
```
Comment (tommylung) [inline data/brands/shop/telecommunication.json]: Missing comma

**Review by tommylung — CHANGES_REQUESTED** [review]

File: data/brands/shop/telecommunication.json [hunk 4]
```diff
@@ -195,6 +207,18 @@
...
       }
     },
+    {
+      "displayName": "Globe",
+      "locationSet": {"include": ["ph"]},
+      "matchNames": ["globe telecom"],
+      "tags": {
+        "alt_name": "Globe Telecom",
+        "brand": "Globe",
+        "brand:wikidata": "Q407009",
```
[the remark this answers is not in the document]
Comment (TagaSanPedroAko) [reply data/brands/shop/telecommunication.json:217]: Yes.

**Comment by TagaSanPedroAko** [comment]
@tommylung Looks good to go.

**Review by tommylung — APPROVED** [review]

**Pull request merged**